======================= Schemathesis test session starts =======================
Schema location: file:///mnt/d/ag/github/lazarus_target_apis/data/api_petstore/api_petstore.json
Base URL: https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0
Specification version: Open API 3.0.0
Random seed: 29564095609858417126394177892864749924
Workers: 1
Collected API operations: 20
Collected API links: 0
API probing: ...API probing: SUCCESS
Schema analysis: ...Schema analysis: SKIP

POST /AAYUSHGARGBU/api_petstore/1.0.0/pet E                               [  5%]
PUT /AAYUSHGARGBU/api_petstore/1.0.0/pet F                                [ 10%]
GET /AAYUSHGARGBU/api_petstore/1.0.0/pet/findByStatus F                   [ 15%]
GET /AAYUSHGARGBU/api_petstore/1.0.0/pet/findByTags F                     [ 20%]
GET /AAYUSHGARGBU/api_petstore/1.0.0/pet/{petId} F                        [ 25%]
POST /AAYUSHGARGBU/api_petstore/1.0.0/pet/{petId} F                       [ 30%]
DELETE /AAYUSHGARGBU/api_petstore/1.0.0/pet/{petId} F                     [ 35%]
POST /AAYUSHGARGBU/api_petstore/1.0.0/pet/{petId}/uploadImage F           [ 40%]
GET /AAYUSHGARGBU/api_petstore/1.0.0/store/inventory F                    [ 45%]
POST /AAYUSHGARGBU/api_petstore/1.0.0/store/order F                       [ 50%]
GET /AAYUSHGARGBU/api_petstore/1.0.0/store/order/{orderId} F              [ 55%]
DELETE /AAYUSHGARGBU/api_petstore/1.0.0/store/order/{orderId} F           [ 60%]
POST /AAYUSHGARGBU/api_petstore/1.0.0/user .                              [ 65%]
POST /AAYUSHGARGBU/api_petstore/1.0.0/user/createWithArray .              [ 70%]
POST /AAYUSHGARGBU/api_petstore/1.0.0/user/createWithList .               [ 75%]
GET /AAYUSHGARGBU/api_petstore/1.0.0/user/login F                         [ 80%]
GET /AAYUSHGARGBU/api_petstore/1.0.0/user/logout .                        [ 85%]
GET /AAYUSHGARGBU/api_petstore/1.0.0/user/{username} F                    [ 90%]
PUT /AAYUSHGARGBU/api_petstore/1.0.0/user/{username} F                    [ 95%]
DELETE /AAYUSHGARGBU/api_petstore/1.0.0/user/{username} F                 [100%]

==================================== ERRORS ====================================
__________________ POST /AAYUSHGARGBU/api_petstore/1.0.0/pet ___________________
Runtime Error

'latin-1' codec can't encode character '\u0100' in position 0: Body ('Ā') is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.

    Traceback (most recent call last):
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/runner/impl/core.py", line 816, in network_test
        response = _network_test(case, *args)
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/runner/impl/core.py", line 848, in _network_test
        response = case.call(**kwargs)
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/models.py", line 310, in call
        response = self.operation.schema.transport.send(
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/transports/__init__.py", line 143, in send
        response = session.request(**data)  # type: ignore
      File "/usr/local/lib/python3.8/dist-packages/requests/sessions.py", line 589, in request
        resp = self.send(prep, **send_kwargs)
      File "/usr/local/lib/python3.8/dist-packages/requests/sessions.py", line 703, in send
        r = adapter.send(request, **kwargs)
      File "/usr/local/lib/python3.8/dist-packages/requests/adapters.py", line 486, in send
        resp = conn.urlopen(
      File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 665, in urlopen
        httplib_response = self._make_request(
      File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 387, in _make_request
        conn.request(method, url, **httplib_request_kw)
      File "/usr/lib/python3.8/http/client.py", line 1256, in request
        self._send_request(method, url, body, headers, encode_chunked)
      File "/usr/lib/python3.8/http/client.py", line 1301, in _send_request
        body = _encode(body, 'body')
      File "/usr/lib/python3.8/http/client.py", line 164, in _encode
        raise UnicodeEncodeError(
    UnicodeEncodeError: 'latin-1' codec can't encode character '\u0100' in position 0: Body ('Ā') is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.

Runtime Error

'latin-1' codec can't encode character '\U00040fbe' in position 0: Body ('\U00040fbe') is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.

    Traceback (most recent call last):
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/runner/impl/core.py", line 816, in network_test
        response = _network_test(case, *args)
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/runner/impl/core.py", line 848, in _network_test
        response = case.call(**kwargs)
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/models.py", line 310, in call
        response = self.operation.schema.transport.send(
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/transports/__init__.py", line 143, in send
        response = session.request(**data)  # type: ignore
      File "/usr/local/lib/python3.8/dist-packages/requests/sessions.py", line 589, in request
        resp = self.send(prep, **send_kwargs)
      File "/usr/local/lib/python3.8/dist-packages/requests/sessions.py", line 703, in send
        r = adapter.send(request, **kwargs)
      File "/usr/local/lib/python3.8/dist-packages/requests/adapters.py", line 486, in send
        resp = conn.urlopen(
      File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 665, in urlopen
        httplib_response = self._make_request(
      File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 387, in _make_request
        conn.request(method, url, **httplib_request_kw)
      File "/usr/lib/python3.8/http/client.py", line 1256, in request
        self._send_request(method, url, body, headers, encode_chunked)
      File "/usr/lib/python3.8/http/client.py", line 1301, in _send_request
        body = _encode(body, 'body')
      File "/usr/lib/python3.8/http/client.py", line 164, in _encode
        raise UnicodeEncodeError(
    UnicodeEncodeError: 'latin-1' codec can't encode character '\U00040fbe' in position 0: Body ('\U00040fbe') is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.

Runtime Error

'latin-1' codec can't encode character '\U00053fca' in position 6: Body ('\U00053fca') is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.

    Traceback (most recent call last):
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/runner/impl/core.py", line 816, in network_test
        response = _network_test(case, *args)
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/runner/impl/core.py", line 848, in _network_test
        response = case.call(**kwargs)
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/models.py", line 310, in call
        response = self.operation.schema.transport.send(
      File "/usr/local/lib/python3.8/dist-packages/schemathesis/transports/__init__.py", line 143, in send
        response = session.request(**data)  # type: ignore
      File "/usr/local/lib/python3.8/dist-packages/requests/sessions.py", line 589, in request
        resp = self.send(prep, **send_kwargs)
      File "/usr/local/lib/python3.8/dist-packages/requests/sessions.py", line 703, in send
        r = adapter.send(request, **kwargs)
      File "/usr/local/lib/python3.8/dist-packages/requests/adapters.py", line 486, in send
        resp = conn.urlopen(
      File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 665, in urlopen
        httplib_response = self._make_request(
      File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 387, in _make_request
        conn.request(method, url, **httplib_request_kw)
      File "/usr/lib/python3.8/http/client.py", line 1256, in request
        self._send_request(method, url, body, headers, encode_chunked)
      File "/usr/lib/python3.8/http/client.py", line 1301, in _send_request
        body = _encode(body, 'body')
      File "/usr/lib/python3.8/http/client.py", line 164, in _encode
        raise UnicodeEncodeError(
    UnicodeEncodeError: 'latin-1' codec can't encode character '\U00053fca' in position 6: Body ('\U00053fca') is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.

Need more help?
    Join our Discord server: https://discord.gg/R9ASRAmHnA
=================================== FAILURES ===================================
__________________ POST /AAYUSHGARGBU/api_petstore/1.0.0/pet ___________________
1. Test Case ID: ITmsWk

- Undocumented HTTP status code

    Received: 429
    Documented: 405

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X POST -H 'Content-Type: application/xml' -d '<Pet>True</Pet>' https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/pet

___________________ PUT /AAYUSHGARGBU/api_petstore/1.0.0/pet ___________________
[FLAKY] Schemathesis was not able to reliably reproduce this failure

1. Test Case ID: hjXmk1

- Undocumented HTTP status code

    Received: 429
    Documented: 400, 404, 405

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X PUT -H 'Content-Type: application/xml' https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/pet

____________ GET /AAYUSHGARGBU/api_petstore/1.0.0/pet/findByStatus _____________
1. Test Case ID: iQmmIg

- Undocumented HTTP status code

    Received: 429
    Documented: 200, 400

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X GET 'https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/pet/findByStatus?status=sold'

_____________ GET /AAYUSHGARGBU/api_petstore/1.0.0/pet/findByTags ______________
1. Test Case ID: eEsuzK

- Undocumented HTTP status code

    Received: 429
    Documented: 200, 400

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X GET 'https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/pet/findByTags?tags=%3D%F1%99%A5%B6%00%C3%A4r%C3%93P%C3%A8%5C&tags=%F4%82%AE%8A%C3%83&tags=&tags=%C3%8A%C3%81U&tags=E%F0%B0%AB%81'

_______________ GET /AAYUSHGARGBU/api_petstore/1.0.0/pet/{petId} _______________
1. Test Case ID: idanfq

- Undocumented HTTP status code

    Received: 429
    Documented: 200, 400, 404

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X GET -H 'api_key: [Filtered]' https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/pet/0

______________ POST /AAYUSHGARGBU/api_petstore/1.0.0/pet/{petId} _______________
1. Test Case ID: fJvABO

- Undocumented HTTP status code

    Received: 429
    Documented: 405

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X POST -H 'Content-Type: application/x-www-form-urlencoded' https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/pet/0

_____________ DELETE /AAYUSHGARGBU/api_petstore/1.0.0/pet/{petId} ______________
1. Test Case ID: aWIZ6J

- Undocumented HTTP status code

    Received: 429
    Documented: 400, 404

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X DELETE https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/pet/0

________ POST /AAYUSHGARGBU/api_petstore/1.0.0/pet/{petId}/uploadImage _________
1. Test Case ID: fIxbFj

- Undocumented HTTP status code

    Received: 429
    Documented: 200

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X POST -H 'Content-Type: application/octet-stream' https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/pet/0/uploadImage

_____________ GET /AAYUSHGARGBU/api_petstore/1.0.0/store/inventory _____________
1. Test Case ID: zKn19p

- Undocumented HTTP status code

    Received: 429
    Documented: 200

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X GET -H 'api_key: [Filtered]' https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/store/inventory

______________ POST /AAYUSHGARGBU/api_petstore/1.0.0/store/order _______________
1. Test Case ID: lqTBjS

- Undocumented HTTP status code

    Received: 429
    Documented: 200, 400

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X POST -H 'Content-Type: application/json' -d '{}' https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/store/order

__________ GET /AAYUSHGARGBU/api_petstore/1.0.0/store/order/{orderId} __________
1. Test Case ID: 54WXSz

- Undocumented HTTP status code

    Received: 429
    Documented: 200, 400, 404

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X GET https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/store/order/1

________ DELETE /AAYUSHGARGBU/api_petstore/1.0.0/store/order/{orderId} _________
[FLAKY] Schemathesis was not able to reliably reproduce this failure

1. Test Case ID: 843gjX

- Undocumented HTTP status code

    Received: 429
    Documented: 400, 404

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X DELETE https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/store/order/1

_______________ GET /AAYUSHGARGBU/api_petstore/1.0.0/user/login ________________
1. Test Case ID: EZIjls

- Undocumented HTTP status code

    Received: 429
    Documented: 200, 400

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X GET 'https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/user/login?password=&username='

_____________ GET /AAYUSHGARGBU/api_petstore/1.0.0/user/{username} _____________
1. Test Case ID: EZWaVZ

- Undocumented HTTP status code

    Received: 429
    Documented: 200, 400, 404

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X GET https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/user/0

_____________ PUT /AAYUSHGARGBU/api_petstore/1.0.0/user/{username} _____________
1. Test Case ID: WCkbti

- Undocumented HTTP status code

    Received: 429
    Documented: 400, 404

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X PUT -H 'Content-Type: application/json' -d '{}' https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/user/0

___________ DELETE /AAYUSHGARGBU/api_petstore/1.0.0/user/{username} ____________
1. Test Case ID: pyJIAE

- Undocumented HTTP status code

    Received: 429
    Documented: 400, 404

[429] Too Many Requests:

    `{"error":"Exceeded rate limit"}`

Reproduce with: 

    curl -X DELETE https://virtserver.swaggerhub.com/AAYUSHGARGBU/api_petstore/1.0.0/user/0

=================================== SUMMARY ====================================

Performed checks:
    not_a_server_error                              404 / 404 passed          PASSED 
    status_code_conformance                         322 / 404 passed          FAILED 
    content_type_conformance                        404 / 404 passed          PASSED 
    response_headers_conformance                    404 / 404 passed          PASSED 
    response_schema_conformance                     404 / 404 passed          PASSED 

Note: Use the 'X-Schemathesis-TestCaseId' header to correlate test case ids from failure messages with server logs for debugging.

Note: To replicate these test failures, rerun with `--hypothesis-seed=29564095609858417126394177892864749924`

Tip: Use the `--report` CLI option to visualize test results via Schemathesis.io.
We run additional conformance checks on reports from public repos.

=================== 4 passed, 16 failed, 1 errored in 67.45s ===================
